src/Data.h

```diff
@@ -23,5 +23,4 @@ QStringList toStringList(const QList<QByteArray>& list);
 QByteArray encodeString(const QByteArray& str, const QString& encoding);
 
 QByteArray decodeString(const QByteArray& str, const QString& encoding);
-
```

Comment:
I think we can merge this PR, since the review done by @MKleusberg seems addressed, but I think it would be better if you removed this leftover of the previous change. We can squash and merge later to get a clean history.

Reply:
I can redo the pull for a clean commit against current master. Easily resolved to make life easier.